frontend/app/src/store/balances/blockchain-balances.ts

```diff
@@ -66,8 +66,49 @@ export const useBlockchainBalancesStore = defineStore(
 
     const loopringBalancesState: Ref<AccountAssetBalances> = ref({});
 
-    const blockchainTotalsState: Ref<AssetBalances> = ref({});
-    const blockchainLiabilitiesState: Ref<AssetBalances> = ref({});
+    const blockchainTotalsState = ref<Record<Blockchain, AssetBalances>>({
```

Comment:
can we change the format to the same as before, the type goes to the var instead? In some cases it helps with a bug with the BigNumber inference, also the state initialization seems to be the same so better extract it to a method, can be also useful with reset

Reply:
For this, I need to separate the total for each chain, since the previous implementation has problems when deleting accounts.

Reply:
Yes, I understand the why but the changes I am requesting are simpler.

`const blockchainTotalsState: Ref<Record<Blockchain, AssetBalances>> = ref(defaultTotals());`

and default totals returns the initial object when called